inst/assets/_book.yml

```diff
@@ -34,15 +34,16 @@ book:
       chapters:
         - pages/taxonomy.qmd
         - pages/wrangling.qmd
+    - part: "QC & preprocessing"
+      chapters:
+        - pages/preprocess.qmd
+        - pages/quality_control.qmd
         - pages/subsetting.qmd
         - pages/agglomeration.qmd
         - pages/transformation.qmd
-    - part: "Exploration & QC"
-      chapters:
-        - pages/quality_control.qmd
-        - pages/composition.qmd
     - part: "Diversity & similarity"
```

Comment:
Note that I have changed the organization of these chapters.

Now there are "QC & preprocessing" topic that holds QC, filtering and transformation. Moreover, I added short introduction to this topic (preprocess.qmd)

Reply:
The idea is to briefly describe the context of the topic. If this is good idea we can use this same idea in other topics also. Or we can remove this preprocess.qmd file and add the information to relevant chapters instead.